edx_proctoring/backends/rest.py

```diff
@@ -198,6 +200,11 @@ def on_review_callback(self, attempt, payload):
         Called when the reviewing 3rd party service posts back the results
         """
         # REST backends should convert the payload into the expected data structure
+        if payload.get('reviewed_by', False):
+            try:
+                payload['reviewed_by'] = User.objects.get(email=payload['reviewed_by'])
```

Comment:
I think this will ultimately fail, because the raw payload is later jsonified and saved to the db in a field in the review, and a User object can't be jsonified.

Perhaps it'd be better to do the user lookup in `views.BaseReviewCallback.make_review`

Reply:
Can we also just swap it out to be the original payload, rather than the backend-translated one? I feel like that would be less radical of a change from what I have, and would get us back to the old shape of how we used to store Software Secure reviews.

Reply:
i.e. https://github.com/edx/edx-proctoring/pull/511/commits/4b910b42da07c11ac429d8d35c62104a57b2304f